contracts/actions/fluid/helpers/FluidRatioHelper.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.24;
+
+import { IERC20 } from "../../../interfaces/IERC20.sol";
+import { IFluidVaultResolver } from "../../../interfaces/fluid/IFluidVaultResolver.sol";
+import { FluidHelper } from "./FluidHelper.sol";
+import { TokenUtils } from "../../../utils/TokenUtils.sol";
+import { DSMath } from "../../../DS/DSMath.sol";
+
+/// @title Helper methods for Fluid ratio calc.
+contract FluidRatioHelper is DSMath, FluidHelper {
```

Comment:
No need for DSMath just for WAD constant, but it isn't very impactful besides a bit lower deployment cost, so just a note.